SampleApps/WebView2_WinUI3_Sample/readme.md

```diff
@@ -0,0 +1,50 @@
+# Introduction
```

Comment:
Add YAML fields, and onboard this sample so it appears at https://docs.microsoft.com/en-us/samples/browse/?terms=webview2

will msg offline how to do this

Reply:
Onboarding the sample is independent from merging this PR.  Closing thread.  Entered a Comment/Suggestion that provides the YAML fields.